src/main/java/athena/exceptions/InputErrorCode.java

```diff
@@ -0,0 +1,14 @@
+package athena.exceptions;
+
+/**
+ * Represents a possible type of user input error when giving commands to Athena.
+ */
+public enum InputErrorCode {
```

Comment:
I like how to create an Enum class for errors! Maybe I will try in my code next time 🤔